### DISCUSSION TITLE
Heatmap: Add toggle to display % of total instead of actual bucket count

### DISCUSSION THREAD

**Discussion by vaizki** [grafana/grafana#70694] [Dashboards, Panels & Visualizations]
Often we are interested not in the total amount of events that fall into a bucket for a timeslot but instead the % events in that timeslot that fall into the bucket. This can be done with a well crafted query (in Prom using division, ignore and group_left etc) but it would be VERY handy if there was simply a toggle in the Heatmap panel that show percentages inside the time slice instead of a absolute values. 

The usage is similiar to the "Stack series: 100%" selection in Time Series panel. 

For example: Web server traffic varies greatly between time of day but I want to visualize (color the cells) based on the % of requests that take more than X ms to complete and not the absolute count of requests.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82662. Feel free to continue the discussion around this there. Thank you!
